Documentation/ceph-cluster-crd.md

```diff
@@ -12,10 +12,12 @@ available for a Ceph cluster.
 Settings can be specified at the global level to apply to the cluster as a whole, while other settings can be specified at more fine-grained levels.  If any setting is unspecified, a suitable default will be used automatically.
 
 ### Cluster metadata
+
```

Comment:
did your editor automatically add these newlines or was that your preference?

Reply:
Yes, as we wrote in private I added the two style points to the community meeting agenda to discuss them if they should be generally used/"enforced".